packages/store/.npmignore

```diff
@@ -9,3 +9,4 @@
 !package.json
 !README.md
 !dist/**
+!cache/solidity-files-cache.json
```

Comment:
small thing but can we pop this up near `out` so all the forge ones are together and maybe leave a comment about why this is here?